lib/ansible/modules/network/nxos/nxos_config.py

```diff
@@ -290,9 +290,12 @@ def get_running_config(module, config=None):
     contents = module.params['running_config']
     if not contents:
         if not module.params['defaults'] and config:
+            # Optimization: The running-config has already been recovered by the backup
```

Comment:
To make it work, `not module.params['defaults']` needs to be removed from this condition.